core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -63,6 +63,7 @@ class Invoker(
     with ActionLogDriver {
 
     private implicit val executionContext: ExecutionContext = actorSystem.dispatcher
+    val invokerName = s"invoker$instance"
```

Comment:
in the constructor, the Invoker extends `MessageHandler(s"invoker$instance")` which provides `this.name`.